content/blog/2020-07-31-spreadsheet-workflows-using-r/index.Rmarkdown

```diff
@@ -0,0 +1,185 @@
+---
+title: Spreadsheets Workflows in R
+date: '2020-07-31'
+slug: spreadsheets-using-r
+categories:
+  - learn
+tags:
+  - spreadsheets
+  - tidyverse
+author:
+  - Devin Johnson
+photo:
+  url: https://unsplash.com/photos/Wpnoqo2plFA
+  author: Mika Baumeister
+---
+
+There's so much to discover and learn when working with spreadsheets, and creating a resource that speaks to both spreadsheet and R users has been an exciting challenge.  While my internship has focused on a plethora of topics from comparing numerous R package functionalities that work with spreafsheets, to writing tips and tricks on best practices for working in both spreadsheets and RStudio, what I'd like to focus on in this post is a section dedicated to the R versions of common spreadsheet workflows.
+
+To do that we'll assume the role of a data analyst in two hypothetical scenarios and walkthrough the tidyverse equivalent of some common analyes and tasks one is likely to come across when working in spreadsheets.   
+
+
+For our first story, its day one as a new data analyst and our boss has asked us to take a look at the company's finances.
```

Comment:
Right around here, I suggest you mention that, of course 😁, the data is available in an Excel workbook.